lib/camerapreview.dart

```diff
@@ -351,27 +360,22 @@ class CameraAwesomeState extends State<CameraAwesome> with WidgetsBindingObserve
     });
   }
 
-  _initAndroidPhotoSize() {
-    if (selectedAndroidPhotoSize == null) {
-      return;
-    }
-    selectedAndroidPhotoSize!.addListener(() async {
-      if (selectedAndroidPhotoSize!.value == null || !Platform.isAndroid) {
+  _initPhotoSize() {
+    widget.photoSize.addListener(() async {
```

Comment:
I won't ask you to refactor this, because there are other listeners added like this as well, but this can be inherently unsafe, because this class doesn't own the `ValueNotifier`s that it adds its listeners to. To be safe, the anonymous/inline functions should be refactored to actual function definitions and added/removed like this:
```
_initPhotoSize() {
  widget.photoSize.addListener(doFoo);
}

@override
dispose() {
  widget.photoSize.removeListener(doFoo);
  super.dispose();
}